.idea/modules.xml

```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Sensorics.iml" filepath="$PROJECT_DIR$/Sensorics.iml" />
+      <module fileurl="file://$PROJECT_DIR$/Sensorics_.iml" filepath="$PROJECT_DIR$/Sensorics_.iml" />
```

Comment:
Why this?

Reply:
I've got no ideal, ill check and remove it

Reply:
of its in the way I cloned it, i named the directory Sensorics_, ill fix it

Reply:
Fixed